docker-compose.yml

```diff
@@ -42,7 +42,37 @@ services:
     volumes:
       - postgres_data:/var/lib/postgresql/data/
 
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    hostname: prometheus
+    user: 65534:65534
+    ports:
+      - 9090:9090
+    volumes:
+      - ./prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+      - prometheus_data:/prometheus_data
+    networks:
+      - monitoring_network
+
+  grafana:
+    image: grafana/grafana:latest
+    container_name: grafana
+    hostname: grafana
+    ports:
+      - 8081:3000
+    volumes:
+      - grafana_data:/var/lib/grafana
+      - ./grafana/provisioning:/etc/grafana/provisioning
+    networks:
+      - monitoring_network
```

Comment:
Might be overkill, but it might be a good idea for sequrity reasons to put grafana <-> prometheus on their own network, such that grafana is not on a shared network with dashboard / backend.